graphql/schema.graphql

```diff
@@ -143,6 +143,12 @@ type Mutation {
   clearMySubscriptions: Int!
 }
 
+input TaskToRestart {
```

Comment:
Should this be PatchesToRestart instead of TaskToRestart, since each TaskToRestart object corresponds to one patch rather than one task? Or something like that; I hesitate to commit to this naming because it's not immediately clear. For example, RestartVersions passes in one version ID but then iterates through taskIds. This is bound to cause confusion in the future. Also taskIds is confusing as a variable name for the object, since there's also an inner taskIds.

Reply:
done